src/main/java/com/artipie/front/settings/RepoPermissions.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2022 artipie.com
+ * https://github.com/artipie/front/LICENSE.txt
+ */
+package com.artipie.front.settings;
+
+import javax.json.JsonArray;
+import javax.json.JsonStructure;
+
+/**
+ * This interface if meant to manage repository permissions.
+ * @since 0.1
+ */
+public interface RepoPermissions {
+
+    /**
+     * Read permissions of the repository.
+     * @param repo Repository name
+     * @return Permissions as json structure
+     * @throws com.artipie.front.api.NotFoundException If repository does not exist
+     */
+    JsonStructure get(String repo);
```

Comment:
Why `JsonStructure` is used here? Don't we know that it's either arrau or object?